aten/src/ATen/native/quantized/cpu/qlinear.cpp

```diff
@@ -900,6 +901,104 @@ at::Tensor PackedLinearWeightsOnednn:: apply_tanh(
       std::move(input), output_scale, output_zero_point);
 }
 
+static at::Tensor linear_int8_with_onednn_weight(
+    at::Tensor input, // int8 CPU Tensor, not QTensor
+    double input_scale,
+    int64_t input_zero_point,
+    at::Tensor onednn_weight, // int8 tensor from MkldnnCPU
+    at::Tensor weight_scales,
+    at::Tensor weight_zero_points,
+    c10::optional<at::Tensor> bias, // plain tensor
+    double output_scale,
```

Comment:
After inductor freezing pass, the `output_scale` from the quant node at output should be reciprocal. Do we need to do the reciprocal here to convert it back?

Reply:
Thanks. It's fixed.